cni/network/network.go

```diff
@@ -11,14 +11,15 @@ import (
 	"net"
 	"net/http"
 	"os"
+	"strconv"
 	"strings"
 	"time"
 
 	"github.com/Azure/azure-container-networking/aitelemetry"
 	"github.com/Azure/azure-container-networking/cni"
 	"github.com/Azure/azure-container-networking/cni/api"
 	"github.com/Azure/azure-container-networking/cns"
-	cnsclient "github.com/Azure/azure-container-networking/cns/client"
+	cnsc "github.com/Azure/azure-container-networking/cns/client"
```

Comment:
import name subject to opinions, no preference here, avoiding collision

Reply:
can be cnscli?

Reply:
cnscli is what i have called the client elsewhere to keep it from conflicting with the cnsclient import